parts/kubernetesmastercustomdata.yml

```diff
@@ -201,8 +201,6 @@ write_files:
     KUBELET_REGISTER_NODE=--register-node=true
     KUBELET_REGISTER_WITH_TAINTS=--register-with-taints={{WrapAsVariable "registerWithTaints"}}
   {{end}}
-{{else}}
```

Comment:
As far as I can tell we still need this to support 1.5 clusters (which the codebase as-is currently still supports).

Reply:
reverted changes that affected v1.5.